src/host/util/iirfilter.h

```diff
@@ -0,0 +1,87 @@
+/******************************************************************************
+* iirfilter.h
+*
+* Project:  KBox
+* Purpose:  Class that implements single order inifinite-impulse-response filter
+* Author:   Transmitterdan, adaptions for KBox by Ronnie Zeiller 2017
+***************************************************************************
+*   Copyright (C) 2016                                                    *
+*                                                                         *
+*   This program is free software; you can redistribute it and/or modify  *
+*   it under the terms of the GNU General Public License as published by  *
```

Comment:
This license is a problem. If we include a single GPL file then the entire project must be distributed under the GPL. This is very unfortunate but I think we have to be careful with this because it would be a shame to put so much effort into the project and get into legal issues later.

I think we should try to find a MIT or Apache licensed library that does the same thing. For example this would probably work well for us: https://github.com/JonHub/Filters